wire-ios-data-model/Tests/OneOnOne/OneOnOneResolverTests.swift

```diff
@@ -257,6 +257,7 @@ final class OneOnOneResolverTests: XCTestCase {
         await viewContext.perform {
             XCTAssertEqual(conversation.messageProtocol, .proteus)
             XCTAssertTrue(conversation.isForcedReadOnly)
+            XCTAssertEqual(conversation.lastMessage?.systemMessageData?.systemMessageType, .mlsNotSupportedSelfUser)
```

Comment:
Perhaps we should also test the case where the other user does not support mls.

Reply:
@johnxnguyen Done!